sdk/keyvault/Azure.Security.KeyVault.Administration/readme.md

```diff
@@ -181,9 +152,20 @@ Content-Length: 143
 Content-Type: application/json
 ```
 
+### Setting up console logging
+The simplest way to see the logs is to enable the console logging.
+To create an Azure SDK log listener that outputs messages to console use AzureEventSourceListener.CreateConsoleLogger method.
+
+```
+// Setup a listener to monitor logged events.
+using AzureEventSourceListener listener = AzureEventSourceListener.CreateConsoleLogger();
+```
+
+To learn more about other logging mechanisms see [here](logging).
+
 ## Next steps
 
-Content forthcoming
+Get started with our [samples](admin_client_samples).
```

Comment:
Link refs have to use square brackets. This won't render right, and will point to non-existent link (literally) `admin_client_samples`.

Reply:
Thanks for catching this 🤦